subprojects/base-services/src/main/java/org/gradle/internal/operations/BuildOperationCategory.java

```diff
@@ -22,14 +22,45 @@
  * @since 4.0
  */
 public enum BuildOperationCategory {
-    CONFIGURE_PROJECT(true, false),
-    TASK(true, true),
+    /**
+     * Configure the root build. May also include nested {@link #CONFIGURE_BUILD} and {@link #RUN_WORK} operations.
+     */
     CONFIGURE_ROOT_BUILD(false, false),
+
+    /**
+     * Configure a nested build or a buildSrc build.
```

Comment:
How can a nested build be within the root build?

Reply:
It can be, but your documentation is still correct:
The hierarchy of build operations can be root build -> nested build -> nested build. So a nested build can be inside of the root build. That still means that any nested builds emits an operation with category `CONFIGURE_BUILD` and not `CONFIGURE_ROOT_BUILD`.